src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -78,6 +78,8 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Day = _library.Append(new DayFunction());
         public static readonly TexlFunction Degrees = _library.Append(new DegreesFunction());
         public static readonly TexlFunction DegreesT = _library.Append(new DegreesTableFunction());
+        public static readonly TexlFunction DEC2HEX = _library.Append(new DEC2HEXFunction());
+        public static readonly TexlFunction DEC2HEXT = _library.Append(new DEC2HEXTFunction());
```

Comment:
>DEC2HEXT [](http://example.com/codeflow?start=44&length=8)

It's safer to not add the function to the `_library` at first, adding them in the section at the end of this file. This way we can have the changes ready in this file without needing to worry about the other runtimes that use it. Take a look at the DateTime function as an example